crypto/init.c

```diff
@@ -684,7 +708,56 @@ int OPENSSL_atexit(void (*handler)(void))
     return 1;
 }
 
-#ifdef OPENSSL_SYS_UNIX
+#ifndef OPENSSL_SYS_UNIX
```

Comment:
Why?

Reply:
Why not?  I had to had unix-only code, and I decided to put the dummy non-unix code first.

Reply:
Never mind, I didn't see the `#else`...  it was too close